tests/x11/thunderbird/thunderbird_common.pm

```diff
@@ -0,0 +1,171 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: send an email using SMTP and receive it using IMAP
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+package thunderbird_common;
+
+use strict;
+use warnings;
+use base "x11test";
+use testapi;
+use utils;
+use Exporter;
+
+our @ISA= qw( Exporter );
+our @EXPORT = qw( tb_setup_account tb_send_message tb_check_email );
+
+sub tb_start {
+    my ($self, $mail_box, $mail_passwd) = @_;
+
+    mouse_hide(1);
+
+    # clean up and start thunderbird
+    x11_start_program("xterm -e \"killall -9 thunderbird; find ~ -name *thunderbird | xargs rm -rf;\"", valid => 0);
```

Comment:
Does a normal terminate not suffice?

Reply:
nope, we need to cleanup any existing config in the homer dir, otherwise we don't have a fresh starting point